src/actions/build/compiler_message_parser.rs

```diff
@@ -0,0 +1,96 @@
+use std::path::PathBuf;

```

Comment:
I'm a bit confused about the module structure here - we have build.rs at the root, plus a complex build submodule of actions. That seems confusing. I assume that actions/build is just the stuff from the build action. I think it would be better just to have this module inside actions, and not have a `build` submodule of actions.